src/translations/en.json

```diff
@@ -17,11 +17,16 @@
   "ActivityFeed.transitionExpire": "Booking request expired. { displayName } did not reply on time.",
   "ActivityFeed.transitionRequest": "{ displayName } requested to book { listingTitle }.",
   "ActivityFeed.transitionReview": "{ displayName } left a review for you. { reviewLink }",
+  "AuthenticationPage.confirmSignupInfoText": "Please check that your information is correct.",
+  "AuthenticationPage.confirmSignupWithIdpTitle": "Signup with {idp}",
```

Comment:
One more comment: This idp variable was in lower case: "Signup with facebook".
So, before you pass the variable to FormattedMessage, you should capitalize it.